src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -385,6 +383,8 @@
   <ItemGroup>
     <Compile Include="$(BclSourcesRoot)\System\Reflection\Assembly.CoreCLR.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyName.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyNameHelpers.cs" />
```

Comment:
@weshaggard These new files live [here](https://github.com/dotnet/corert/tree/893cbdc5cf61089085ebf0d7a7a81fc54495020f/src/System.Private.CoreLib/src/System/Reflection/Runtime/Assemblies) in CoreRT. Should I keep the same structure?